opentreemap/treemap/views/tree.py

```diff
@@ -118,6 +120,49 @@ def search_tree_benefits(request, instance):
     return context
 
 
+def simplified_tree_benefits(request, instance):
+    unit = 'kg/rok'
+    req = requests.get(settings.ECO_SERVICE_URL)
+    req.raise_for_status()
+    data = req.json()
+
+    descriptions = {
+        'O3': u'Redukcja ozonu (O3)',
+        'NO2': u'Redukcja dwutlenku azotu (NO2)',
+        'SO2': u'Redukcja dwutlenku siarki (SO2)',
+        'PM2.5': u'Redukcja pyłów 2.5 µm (PM2,5)'
+    }
+
+    benefits = {
+        k.replace('.', ''):
+        {
+            'label': descriptions[k],
+            'value': round(v / 1000, 2),
+            'unit': unit,
+        }
+        for k, v in data['benefits'].iteritems()
+    }
+    benefits['totals'] = {
+        'label': u'Redukcja zanieczyszczeń w sumie',
+        'value': reduce(lambda x, row: x + row['value'], benefits.values(), 0),
+        'unit': unit
+    }
+
+    context = {
+        'used_trees': data['trees_count'],
+        'total_trees': Tree.objects.filter(instance=instance).count(),
+        'n_empty_plots': 0,
```

Comment:
What is `n_empty_plots` and why do we need to set it to 0? Why do we need `n_resources` set to 0? Could that be that setting `has_resources` to False would make these fields not longer required?

Reply:
I decided to keep those entries in the context dict so there is no need to modify html templates. However I can try to remove most of them.